File: 프로그래머스/0/181861. 배열의 원소만큼 추가하기/배열의 원소만큼 추가하기.js
function solution(arr) {
    var answer = [];
    let sum=0;
    for(let i=0; i<arr.length;i++){
            sum+=arr[i];
        }
    let n=0;
    var answer = [sum];
    for(let i=0; i<arr.length;i++){
        for(let j=0; j<arr[i] ;j++){
            answer[n]=arr[i];
            n++;
        }
    }
    return answer;
}

File: 프로그래머스/0/181861. 배열의 원소만큼 추가하기/배열의 원소만큼 추가하기.cs
using System;

public class Solution {
    public int[] solution(int[] arr) {
        int sum=0;
        for(int i=0; i<arr.Length;i++){
            sum+=arr[i];
        }
        int n=0;
        int[] answer =new int[sum];
        for(int i=0; i<arr.Length;i++){
            for(int j=0; j<arr[i] ;j++){
                answer[n]=arr[i];
                n++;
            }
        }
        return answer;
    }
}